frontend/src/console/components/RedeploymentToolbar/RedeploymentToolbar.tsx

```diff
@@ -146,6 +146,23 @@ export const RedeploymentToolbar: React.FC<Props> = ({
           </Button>
         </Modal.Footer>
       </Modal>
+      <div className="toast-container">
+        <Toast
+          className="deployToast"
+          show={toastIsVisible}
+          onClose={toggleToast}
+          delay={4000}
+          autohide
+        >
+          <Toast.Header
+            className="toastHeader"
+            closeLabel={messages.closeToast.defaultMessage}
+          />
+          <Toast.Body>
+            <WrappedMessage id="toastMessage" messages={messages} />
+          </Toast.Body>
+        </Toast>
+      </div>
```

Comment:
Can we move this out in a component? This can be something we might want to reuse. We can make the delay configurable and pass it as a prop, we can do the same for messages like header and body. It will make the components more maintainable.

Reply:
Addressed!